spec/factories/shipments.rb

```diff
@@ -1,6 +1,7 @@
 FactoryBot.define do
   factory :shipment do
-    sub { "uuid" }
     hubee_folder_id { "folder_uuid" }
+    reference { "AAAAAAAAAAAAA" }
```

Comment:
je ferai une séquence ici, ta reference ayant une contrainte d'unicité ta factory va planter dès le 2eme create

```suggestion
    reference
```

et plus haut:

```ruby
sequence(:reference) { |n| n.to_s }
```

(btw ici je vois qu'il n'y a aucune contrainte de longueur d'où ma dummy sequence)

Reply:
Je vais ajouter une contrainte de longueur tiens